src/planning/plan/planificator.ts

```diff
@@ -192,7 +192,7 @@ export class Planificator {
   }
 
   async _storeSearch(): Promise<void> {
-    const handle = handleForActiveStore(this.searchStore, this.arc);
+    const handle = handleForActiveStore(this.searchStore.storeInfo, this.searchStore, this.arc);
```

Comment:
Is there any way to avoid having to pass both the store and store.storeInfo? Seems like a recipe for problems later.

Reply:
this is an incremental step towards not passing the store itself, but the DirectStoreManager instead (and the storeInfo)